pkg/controller/template/render.go

```diff
@@ -370,8 +370,10 @@ func cloudProvider(cfg RenderConfig) (interface{}, error) {
 	// FIXME Explicitly disable (remove) the cloud provider on OpenStack for now
 	// Don't forget to turn the test case back on as well
 	switch cfg.Platform {
-	case platformAWS, platformAzure, platformGCP, platformVSphere:
+	case platformAWS, platformAzure, platformVSphere:
 		return cfg.Platform, nil
+	case platformGCP:
+		return "gce", nil
```

Comment:
just out of curiosity, is there a const defined somewhere for this?

Reply:
https://github.com/kubernetes/kubernetes/blob/844a0c9a10cd52367ec7c05f5a8c176535dcef38/staging/src/k8s.io/legacy-cloud-providers/gce/gce.go#L59

Reply:
Would wanna bring kubernetes into vendor for const :p

Reply:
fair enough then :P